changelog.md

```diff
@@ -4,6 +4,10 @@
 Also see the [Architect functions changelog](https://github.com/arc-repos/arc-functions/blob/master/changelog.md).
 ---
 
+## `master` branch
+
+- [dev] Added test run watcher script.
```

Comment:
not sure if we even want to be including dev stuff in the changelog - lemme know, i can remove

Reply:
👍